src/ui/edit/machine-properties-dialog.c

```diff
@@ -1609,8 +1609,9 @@ make_int_range_widget (const BtMachinePropertiesDialog * self,
   gtk_range_set_value (GTK_RANGE (widget), value);
   // TODO(ensonic): add numerical entry as well ?
 
-  signal_name = g_alloca (9 + strlen (property->name));
-  g_sprintf (signal_name, "notify::%s", property->name);
+  const gint len = 9 + strlen (property->name);
+  signal_name = g_alloca (len);
+  g_snprintf (signal_name, len, "notify::%s", property->name);
```

Comment:
those are safe. Lets only change the code where a number might overflow the format.